### PR TITLE
Refs #21286 -- Enabled ImageField data in serializer data tests.

### PR DESCRIPTION
**Pull request by adamzap**
This patch tests ImageField in the serializer data tests in the same way that FileField is tested. This includes one commented-out test.

The commented-out test is a good addition because it implies that because ImageField is a subclass of FileField, the tests are commented-out for the same reason. They can both be enabled when ticket 10244 is resolved.

#### Trac ticket number

ticket-21286

#### Branch description

This patch tests ImageField in the serializer data tests in the same way that FileField is tested. This includes one commented-out test.

The commented-out test is a good addition because it implies that because ImageField is a subclass of FileField, the tests are commented-out for the same reason. They can both be enabled when ticket 10244 is resolved.

#### Checklist

- [x] This PR targets the `main` branch.
- [x] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [x] I have checked the "Has patch" ticket flag in the Trac system.
- [x] I have added or updated relevant tests.
- [x] I have added or updated relevant docs, including release notes if applicable.
- [x] I have attached screenshots in both light and dark modes for any UI changes.

### CODE REVIEW

File: tests/serializers/test_data.py [hunk 1]
```diff
@@ -291,7 +292,9 @@ def inherited_compare(testcase, pk, klass, data):
     (data_obj, 81, IntegerData, -123456789),
     (data_obj, 82, IntegerData, 0),
     (data_obj, 83, IntegerData, None),
-    # (XX, ImageData
+    (data_obj, 86, ImageData, "file:///foo/bar/whiz.png"),
+    # (data_obj, 87, ImageData, None),
+    (data_obj, 88, ImageData, ""),
```
Comment (adamzap) [inline tests/serializers/test_data.py:297]: See lines 278-280 for where this pattern originates with `FileData`.

**Review by sarahboyce — APPROVED** [review]
Thank you @adamzap 

Note that we could chose to remove the final commented out tests due to the ticket-10244 dependency or close the ticket and add a comment to ticket-10244 if we want to keep them :thinking:

**Comment by adamzap** [comment]
@sarahboyce What is your preference? I hope to start working on [ticket-10244](https://code.djangoproject.com/ticket/10244) soon.

**Comment by sarahboyce** [comment]
> @sarahboyce What is your preference? I hope to start working on [ticket-10244](https://code.djangoproject.com/ticket/10244) soon.

If you're working on ticket-10244 let's leave it like this for now :+1:

**Pull request merged**